h2o-py/h2o/explanation/_explain.py

```diff
@@ -4,14 +4,14 @@
 import warnings
 from contextlib import contextmanager
 from collections import OrderedDict, Counter, defaultdict
+from h2o.plot import decorate_plot_result, get_matplotlib_pyplot
```

Comment:
also, sorry for being annoying, let's group `h2o` imports at least.
Usually the structure of imports looks like this, from the most generic available to the most specific:
```python
# imports from Py standard lib

# imports from 3rd party libraries

# imports from your project (here h2o)

# local relative imports (from .x import …)
```
not asking you to fix all existing modules, but just for your new imports.

(Note that we use some exceptions in this project, like the `__future__` and other custom `h2o.compat` import for Py2, which are always on top, but it's mainly to get rid of it quickly…